src/rust/engine/fs/store/src/snapshot_ops.rs

```diff
@@ -227,6 +238,7 @@ pub trait SnapshotOps: Clone + Send + Sync + 'static {
       directory::Entry::File(f) => {
         files.insert(path.to_owned(), f.digest());
       }
+      directory::Entry::Symlink(_) => todo!(),
```

Comment:
Any change here depends on what (if any) changes are needed to the signature of `DigestTrie::from_path_stats` to support symlinks.